techniques/css/C31.html

```diff
@@ -12,86 +12,84 @@
-               This is done by using layout techniques that adapt to the available viewport space.
+            <h2>解説</h2>
+            <p>この達成方法の目的は、320 CSSピクセルに相当する幅の水平スクロールバー、
+              または水平スクロールするテキストの256 CSSピクセルに相当する高さの垂直スクロールバーを導入せずに
+              コンテンツを表示できるようにすることである。
+              これには、使用可能な表示域の空間に適応するレイアウト手法が使用される。
             </p>
-            <p><a href="https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Flexible_Box_Layout">Flexbox layouts</a> define layout regions that reflow as needed to display the region on the screen.
-               Although the exact layout therefore varies, the relationship of elements and the reading
-               order remains the same when done right. This is an effective way to create designs
-               that present well on different devices and for users with different zoom preferences.
+            <p><a href="https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Flexible_Box_Layout">Flexbox レイアウト</a>は、
+              画面に領域を表示するために必要に応じてリフローするレイアウト領域を定義する。
+              リフローが正しく行われた場合、厳密なレイアウトは異なるが要素の関係と読み取り順序は同じままである。
+              これは、さまざまなデバイスやさまざまなズーム設定を持つユーザーに適したデザインを作成するための効果的な方法である。
             </p>
-            <p>The basic principles of flexbox layouts are to:</p>
+            <p>Flexbox レイアウトの基本原則は以下のとおり:</p>
             <ol>
-               
-               <li>Define the size of layout regions using flexbox properties and media queries for specific
-                  viewport sizes, so they enlarge, shrink or wrap in the available space and respond
-                  to zoom levels;
-               </li>       
-               
-               <li>Position the layout regions in the flexbox container as a row of adjacent flexbox
-                  items, which may wrap to new rows as needed in much the same way as words in a paragraph
-                  wrap.
+
+               <li>Flexboxプロパティとメディアクエリを使用して
+                 特定の表示域サイズのためにレイアウト領域のサイズを定義し、
+                 利用可能なスペースで拡大、縮小、または折り返して、ズームレベルに対応する。
+               </li>
+
+               <li>レイアウト領域を、隣接するFlexboxアイテムの行並びでFlexboxコンテナ内に配置する。
+                 これは、段落の折り返しの単語とほぼ同じ方法で、必要に応じて新しい行に折り返すことができる。
                </li>
-               
+
             </ol>
             <div class="note">
-               <div role="heading" class="note-title marker" aria-level="3">Note</div>
-               <p>Flexbox has the possibility to cause a keyboard navigation disconnect by using the
-                  order and reverse properties. The <a href="https://www.w3.org/TR/css-flexbox-1/#order-accessibility">CSS Flexible Box Layout module warns</a> against resequencing content logic as they cause incorrect source ordering and are
-                  non-conforming.
+               <div role="heading" class="note-title marker" aria-level="3">注記</div>
+               <p>Flexboxでは、orderプロパティとreverseプロパティを使うことでキーボードナビゲーションの切断を引き起こす可能性がある。
```

Comment:
意訳気味ですが、「分断」はどうでしょうか。

```suggestion
               <p>Flexbox では、order プロパティ及び reverse プロパティを使うことでキーボードナビゲーションの分断を引き起こす可能性がある。
```